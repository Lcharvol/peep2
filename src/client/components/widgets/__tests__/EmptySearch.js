import React from 'react';
import 'jest-styled-components';
import { shallow } from 'enzyme'; // eslint-disable-line
import EmptySearch from '../EmptySearch';

describe('<EmptySearch />', () => {
  test('should render an <EmptySearch />', () => {
    const tree = shallow(<EmptySearch />);
    expect(tree).toMatchSnapshot();
  });
  test('should <EmptySearch /> to be flex', () => {
    const tree = shallow(<EmptySearch />);
    expect(tree).toHaveStyleRule('display', 'flex');
  });
});
