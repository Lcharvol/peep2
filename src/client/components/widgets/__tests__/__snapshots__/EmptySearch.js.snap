// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptySearch /> should render an <EmptySearch /> 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <span
    className="sc-bdVaJa iejKxA"
/>,
  "nodes": Array [
    <span
      className="sc-bdVaJa iejKxA"
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_context": Object {},
    "_element": <styled.span />,
    "_instance": StyledComponent {
      "attrs": Object {},
      "context": Object {},
      "props": Object {},
      "refs": Object {},
      "state": Object {
        "generatedClassName": "iejKxA",
        "theme": null,
      },
      "unsubscribeId": -1,
      "updater": Updater {
        "_renderer": [Circular],
      },
    },
    "_newState": Object {
      "generatedClassName": "iejKxA",
      "theme": null,
    },
    "_rendered": <span
      className="sc-bdVaJa iejKxA"
/>,
    "_rendering": false,
    "_updater": Updater {
      "_renderer": [Circular],
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <styled.span />,
}
`;
